$MirOS: ports/mailnews/pine/patches/patch-pine_pine-use_c,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- pine/pine-use.c.orig	Wed Feb 16 23:52:46 2000
+++ pine/pine-use.c	Wed May  4 23:28:11 2005
@@ -98,7 +98,7 @@ main(argc, argv)
         if(strcmp(pw->pw_dir, "/") == 0)
           continue;
 
-        sprintf(filename, "%s/.pinerc", pw->pw_dir);
+        sprintf(filename, "%s/.etc/pinerc", pw->pw_dir);
         if(stat(filename, &statb) < 0)
           continue;
         if(statb.st_mtime + 7 * DAYSEC > now) 
